packages/hardhat-plugin/src/utils/shouldBeHardhatPluginError.ts

```diff
@@ -9,9 +9,9 @@ import type { IgnitionError } from "@nomicfoundation/ignition-core";
  *    - If there's an exception that doesn't fit in either category, let's discuss it and review the categories.
  */
 const whitelist = [
-  200, 201, 202, 203, 403, 404, 405, 600, 601, 602, 700, 701, 702, 703, 704,
-  705, 706, 707, 708, 709, 710, 711, 712, 713, 714, 715, 716, 717, 718, 719,
-  720, 721, 722, 723, 724, 725, 726, 800, 900,
+  200, 201, 202, 203, 204, 403, 404, 405, 600, 601, 602, 700, 701, 702, 703,
```

Comment:
I included it in the whitelist to get the nicer error message:

![image](https://github.com/NomicFoundation/hardhat-ignition/assets/24030/17d4770b-5d56-4706-a615-98dd5b1dcbf0)